src/polyfills/custom-elements.js

```diff
@@ -106,7 +106,8 @@ function isPatched(win) {
  * @param {!Error} error
  */
 function rethrowAsync(error) {
-  new /*OK*/ Promise(() => {
+  setTimeout(() => {
+    self.__AMP_REPORT_ERROR(error);
```

Comment:
Do we know for sure that `setReportError()` will be called by this point?

Reply:
Yes, this can't happen until an AMP element has been created, which happens after the log reporter is setup.